src/malloc_wrapper.cc

```diff
@@ -46,7 +46,7 @@ void gnu_backtrace()
 	}
 
 	for (j = 0; j < nptrs; j++)
-		printf("%s\n", strings[j]);
+		printf("callback: %s\n", strings[j]);
```

Comment:
[llvmlibc-callee-namespace]: 'info<unsigned long, void *>' must resolve to a function declared within the '__llvm_libc' namespace